src/caveats.ts

```diff
@@ -1,19 +1,19 @@
 import { JsonRpcMiddleware } from 'json-rpc-engine';
 import { isSubset } from './@types/is-subset';
-import { IOcapLdCaveat } from './@types/ocap-ld'
+import { IOcapLdCaveat } from './@types/ocap-ld';
 import { unauthorized } from './errors';
-const isSubset = require('is-subset');
+const isSubset = require('is-subset'); // eslint-disable-line
```

Comment:
It complains if an import statement is used.

Reply:
when you mix import and require statements in a file it can confuse bundlers (like rollup) https://github.com/rollup/rollup-plugin-commonjs/issues/273

Reply:
Ah, brutal. This has not been a problem for us, thus far, but it would be great to fix. There must be something I'm missing; we declare the types for that module in `src/@types` and import them 3 lines above the `require` statement, but if I switch to an `import`, it does this:
```
Could not find a declaration file for module 'is-subset'. '/Users/rekmarks/workspaces/ethereum/rpc-cap/node_modules/is-subset/index.js' implicitly has an 'any' type.
  Try `npm install @types/is-subset` if it exists or add a new declaration (.d.ts) file containing `declare module 'is-subset';`ts(7016)
```

Not the first time I've come across this. Please do let me know if you know how to deal with these issues!